### PR TITLE
[NodeBundle] Fix page edit tab logic

### PR DESCRIPTION
**Pull request by jockri**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | 

* When you click on a tab, the url changes (?currenttab=menu) [new feature]
* When you have the tab in the url, the tab will be activated [this was broken]
* When you submit the form, when the page refreshes, the tab will be opened again [this was broken]

### CODE REVIEW

**Comment by roderik** [comment]
@diskwriter js,ok?

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_auto-collapse-tabs.js [hunk 1]
```diff
@@ -18,6 +21,44 @@ kunstmaanbundles.autoCollapseTabs = (function($, window, undefined) {
         }); // when window is resized
     };
 
+    initTabLogic = function () {
+        // If there is a tab defined in the url, we activate it
+        if ($('#currenttab') && $('#currenttab').val().length > 0) {
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_auto-collapse-tabs.js]: Maybe save $('#currenttab') to a var so it does not have to be re-selected all the time.

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_auto-collapse-tabs.js [hunk 2]
```diff
@@ -18,6 +21,44 @@ kunstmaanbundles.autoCollapseTabs = (function($, window, undefined) {
...
+            $('.js-auto-collapse-tabs.nav-tabs a[href="' + $('#currenttab').val() + '"]').tab('show');
+        }
+
+        // When tab click, add the current tab in the url
+        $('.js-auto-collapse-tabs.nav-tabs a').click(function (e) {
+            $(this).tab('show');
+
+            var activeTab = this.hash.substr(1);
+            window.history.pushState({}, null, replaceUrlParam(window.location.href, 'currenttab', activeTab));
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_auto-collapse-tabs.js]: history.pushState is not supported in ie9. I'll look into another way of achieving this.

**Comment by jockri** [comment]
I have added a check that `history.pushState` is supported or not.
On IE9, when you click on a tab, the url will not be change. If you submit the form, the correct tab will still be activated.

**Pull request merged**